dags/glam_fog_release.py

```diff
@@ -78,7 +78,7 @@
         ) as histogram_bucket_counts:
             prev_task = None
             # Windows + Release data is in [0-9] so we're further splitting that range.
-            for sample_range in ([0, 0], [1, 2], [3, 5], [6, 9], [10, 49], [50, 99]):
+            for sample_range in ([0, 0], [1, 1], [2, 3], [4, 6], [7, 9], [10, 49], [50, 99]):
```

Comment:
Do sample ids 0 and 1 have more samples than 2-9?  Just wondering why they're 1% and the others are 2-3%

Reply:
The size is the same for each sample from 0-9 inclusive (1.17TB) but I don't know why the first jobs take longer...